waspc/src/Wasp/Util.hs

```diff
@@ -12,16 +18,25 @@ module Wasp.Util
     leftPad,
     (<++>),
     (<:>),
+    bytestringToHex,
+    Hex (..), -- TODO(shayne): Should we put this into Internal? Or make hexFromString smart constructor?
```

Comment:
That's a good question. Maybe `hexToString` is a way to go for now! And you would expose `Hex` but not `Hex (..)`.

Reply:
I just lifted this from our pairing :D but yeah I can do `hexToString` and just expose that and `Hex` for now!